### PR TITLE
Maintenance Commit w/ Small Changes

### PR DESCRIPTION
**Pull request by mxttwoods**
- ```eslint --fix```
- spell check
- vscode extensions list
- editorconfig
- dependabot tooling
- fix readme

### CODE REVIEW

**Comment by mxttwoods** [comment]
Cheers!

**Comment by mxttwoods** [comment]
Also,

Your killing it on YouTube.

Keep up the good work!

**Comment by FilledStacks** [comment]
Thanks. I'm so happy about the involvement from the backend community. I'll merge this in when I'm back on the open source work. Thanks so much for all the efforts. I really appreciate it.

File: src/functionParser.ts [hunk 1]
```diff
@@ -59,23 +59,27 @@ export class FunctionParser {
...
+    const functionFiles: any = glob.sync(`${this.rootPath}/**/*.function.js`, {
       cwd: this.rootPath,
       ignore: './node_modules/**',
     });
 
     for (let i = 0, fl = functionFiles.length; i < fl; i++) {
-      const file = functionFiles[i];
-      const filePath = parse(file);
-      const directories = filePath.dir.split('/');
+      /** @type {*} */
```
Comment (FilledStacks) [inline src/functionParser.ts:68]: Hey, I don't like having these comments in between the source code for local variables. What exactly does this do for us?

**Review by FilledStacks — CHANGES_REQUESTED** [review]
Thanks! 

Biggest concern for me is the `/** @type {*} */` comments for the vars in the functions. I don't think they are necessary. They don't provide a lot of additional help when working in this package. 

Any specific reason they were added?

**Comment by mxttwoods** [comment]
It's just a quick type reference my editor generated. Happy to clean those up as your right; it doesn't provide any info other than {any} which is inherit.

File: src/functionParser.ts (hunk 1 above)
Comment (mxttwoods) [reply src/functionParser.ts]: no problem! cleaned it up

**Comment by mxttwoods** [comment]
Have a look now! Thanks for the feedback.

**Comment by mxttwoods** [comment]
Sorry I know that's a lot to review, hoping to see this project go 1.0.0 soon! 

Cheers

**Comment by mxttwoods** [comment]
Let me know if you need another commit to merge properly.

**Comment by FilledStacks** [comment]
Thanks @mxttwoods much appreciated!

**Comment by mxttwoods** [comment]
> Thanks @mxttwoods much appreciated!

i'll try and get ci/cd going this week. 

Cheers

**Pull request merged**